.github/workflows/release.yaml

```diff
@@ -25,6 +25,14 @@ jobs:
       - name: Check version
         run: |
           cat VERSION.txt | grep ${{ github.ref_name }}
+      - name: Preflight test
+        run: |
+          echo "::group::E2E Tests"
+          make test-e2e
+          echo "::endgroup::"
+          echo "::group::Integration Tests"
+          OCM_BUNDLE_VERSION=$(cat VERSION.txt) make test-integration
```

Comment:
when I try to run `OCM_BUNDLE_VERSION=$(cat VERSION.txt) make test-integration` locally, I got some errors `couldn't find the requested version bundle: 0.8.0`

Reply:
Ah, you're right--I'll take this out. I wasn't thinking about the VERSION being clusteradm and not OCM. Since this uses the "default" version now these tests would run in the PR when the OCM version is updated, so I'm not so sure a preflight check is even necessary. We'll add some additional E2E tests to cover the managed cluster add-on availability in the future.

Reply:
Updated